libs/components/indicators/src/lib/modules/shared/indicator-description-type.ts

```diff
@@ -1,7 +1,6 @@
 export type SkyIndicatorDescriptionType =
   | 'attention'
   | 'caution'
-  | 'completed'
```

Comment:
We have released the other instances of this. So wouldn't this be a breaking change?